content/docs/strict-mode.md

```diff
@@ -1,65 +1,65 @@
 ---
 id: strict-mode
-title: Strict Mode
+title: Modo Estrito
```

Comment:
`strict mode` is a known text, could you put it in parenthesis for reference as you did below?

```
Modo Estrito (Strict Mode)
```